lib/Doctrine/Common/DataFixtures/Executor/AbstractExecutor.php

```diff
@@ -140,5 +139,3 @@ public function purge()
      */
     abstract public function execute(array $fixtures, $append = false);
 }
-
-interface_exists(ObjectManager::class);
```

Comment:
IIRC the compatibility layer has to stay, care to weigh in @greg0ire?

Reply:
if you keep support for doctrine/persistence 1.x (which has the class name BC layer), yes